Library/docs/source/learn/physical_digital_realities/anchoring.md

```diff
@@ -0,0 +1,357 @@
+   This query is useful when an anchor is associated with specific digital content stored in an asset collection.
+
+   ```
+   AnchorSystem->GetAnchorsByAssetCollectionId("assetCollectionIdABC", 0, 10, [](const AnchorCollectionResult& Result) {
+       // Process retrieved anchors by asset collection
+   });
+   ```
+
+### Updating Anchors
+
+In some scenarios, updating an anchor's attributes may be necessary, such as modifying its digital transform or reassigning it to a different space. Although CSP does not currently support direct updates to anchor attributes, developers can delete an existing anchor and create a new one with the updated parameters to achieve the same result.
+
+### Deleting Anchors
+
+CSP allows the deletion of anchors using the `DeleteAnchors` method. This operation enables developers to manage obsolete or irrelevant anchors effectively.
+
+**Code Example**
+
+```
+// The set of anchors that the client application wishes to delete.
+csp::common::Array<csp::common::String> AnchorIDsToDelete(1) = {"66338b7fcd13d3f163e38c25"};
+
+// The callback to respond to the completion of the anchor delete operation.                        
+csp::systems::NullResultCallback Callback = [](const NullResult& Result)
+{
+    if(Result.GetResultCode() == csp::systems::EResultCode::Success)
+    {
+        // Application logic goes here.
+    };
+};
+                          
+csp::systems::AnchorSystem* AnchorSystem = csp::systems::SystemsManager::Get().GetAnchorSystem();
+AnchorSystem->DeleteAnchors(AnchorIDsToDelete, Callback);
+```
+
+This code deletes a specified anchor and provides a callback to handle the result. If successful, the anchor will no longer be accessible or queryable in CSP.
+
+## Querying for Anchors
+
+Querying anchors in CSP allows developers to locate and manage anchors based on specific criteria. CSP provides methods to query anchors by location, space, or asset collection, enabling flexible and precise retrieval of anchor data.
+
+Anchors can be retrieved via CSP using three different methods in the Anchor System: `GetAnchorsInArea`, `GetAnchorsInSpace`, and `GetAnchorsByAssetCollectionId`.
+
+All of the anchor querying methods support paging. Each parameter allows the application to specify how many anchors to **Skip are** returning and a **Limit** to how many per invocation should be returned.
+
+### GetAnchorsInArea
+
+You can use the `GetAnchorsInArea` method to find anchors within a specified radius of a geographic location.
+
+**Code Example**  
+This example searches for anchors around Los Angeles within a 100-meter radius. The callback handles the retrieved anchors, allowing developers to use them in their applications.
+
+```
+csp::systems::AnchorSystem* AnchorSystem = csp::systems::SystemsManager::Get().GetAnchorSystem();
+
+AnchorSystem->GetAnchorsInArea(
+    csp::systems::GeoLocation(34.0549, -118.2426), // Los Angeles location
+    100.0, // Radius in meters
```

Comment:
[question] while I totally understand why using the metric system makes a lot of sense, will the majority of the current customer base (are they mostly in the US?) be comfortable with it?

not to be resolved or answered now at all. just a UX consideration